hack/release.sh

```diff
@@ -91,9 +85,40 @@ ko resolve ${KO_FLAGS} -R -f config/monitoring/100-common \
     -f config/monitoring/200-common/100-istio.yaml \
     -f config/monitoring/200-common/200-prometheus-exporter \
     -f config/monitoring/200-common/300-prometheus \
-    -f config/monitoring/200-common/100-istio.yaml >> ${LITE_YAML}
+    -f config/monitoring/200-common/100-istio.yaml >> "${MONITORING_LEAN_YAML}"
+
+
+echo "Building Release Bundles."
+
+# These are the "bundled" yaml files that we publish.
+# Local generated yaml file.
+readonly OUTPUT_YAML=release.yaml
```

Comment:
consider renaming `OUTPUT_YAML` to `RELEASE_YAML` since there are so many yamls that out output flying around.

Reply:
Done.